components/script/task_source/mod.rs

```diff
@@ -18,16 +18,16 @@ pub mod websocket;
 
 use std::result::Result;
 
-use enum_iterator::IntoEnumIterator;
-
 use crate::dom::globalscope::GlobalScope;
 use crate::task::{TaskCanceller, TaskOnce};
 
-// The names of all task sources, used to differentiate TaskCancellers.
-// Note: When adding a task source, update this enum.
-// Note: The HistoryTraversalTaskSource is not part of this,
-// because it doesn't implement TaskSource.
-#[derive(Clone, Eq, Hash, IntoEnumIterator, JSTraceable, PartialEq)]
+/// The names of all task sources, used to differentiate TaskCancellers. Note: When adding a task
+/// source, update this enum. Note: The HistoryTraversalTaskSource is not part of this, because it
```

Comment:
Not related to the changes in this PR, but is this comment outdated? The note says HistoryTraversalTaskSource is not part of the enum, but it does have the `HistoryTraversal` variant.

Reply:
Interesting. Looks like the `HistoryTraversal` variant was introduced at the same time (671627e97e20ae4baf728ae6dda61ef6f857c193) as the enum so the comment may have always been wrong? @gterzian, any idea what might be up here?